src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -274,9 +274,10 @@ export class BrowsingContextImpl {
         sandbox,
       });
     }
-    if (maybeSandboxes.length !== 1) {
```

Comment:
should not we still throw an error for the case `maybeSandboxes.length === 0`?

Reply:
We create one if there isn't one in that case.

Reply:
where does it happen? Let's change the type to return Realm|undefined then?

Reply:
I think we should still keep the check for `maybeSandboxes.length === 0` as we are relying on an Event that may or may not have been dispatched (see comment a few line above)

Reply:
> I think we should still keep the check for `maybeSandboxes.length === 0` as we are relying on an Event that may or may not have been dispatched (see comment a few line above)

Ah, yes this is possible. I'll make it an assertion then.